### PR TITLE
Feature/issue 15 add tests

### PR DESCRIPTION
**Pull request by LyndsayLawrence**
Tests for controllers added

### CODE REVIEW

**Comment by LyndsayLawrence** [comment]
Ready for review

File: grails-app/controllers/mama/ng/scheduler/UserController.groovy [hunk 1]
```diff
@@ -18,43 +18,34 @@ class UserController {
...
+            return
         }
+
+        response.status = 200
+        render instance as JSON
     }
 
     def create(User instance) {
+        instance.save()
+        println(instance)
```
Comment (hodgestar) [inline grails-app/controllers/mama/ng/scheduler/UserController.groovy]: Stray print. :)

**Comment by hodgestar** [comment]
:+1: on the commits that aren't mine.

@LyndsayLawrence if you're happy with my commits and Travis tests pass, we can land.

**Comment by LyndsayLawrence** [comment]
:+1:

**Pull request merged**